backend/djangoindia/db/models/event.py

```diff
@@ -89,3 +89,42 @@ def __str__(self) -> str:
         return (
             f"{self.first_name} {self.last_name} ({self.email}) --- {self.event.name}"
         )
+
+    
+
+class SponsorData(BaseModel):

```

Comment:
lets change it to`Sponsor`.
```suggestion
class Sponsor(BaseModel):

```